mappings/net/minecraft/client/render/entity/EndCrystalEntityRenderer.mapping

```diff
@@ -1,10 +1,10 @@
-CLASS net/minecraft/class_892 net/minecraft/client/render/entity/EnderCrystalEntityRenderer
-	FIELD field_21002 SINE_FOURTY_FIVE_DEGREES F
+CLASS net/minecraft/class_892 net/minecraft/client/render/entity/EndCrystalEntityRenderer
+	FIELD field_21002 SINE_45_DEGREES F
```

Comment:
```suggestion
	FIELD field_21002 HALF_SQRT_2 F
```
?

Reply:
![](https://user-images.githubusercontent.com/6596629/76434157-fb0b1e00-63bd-11ea-96f8-8ddd4b42645f.png)
That one is actually calculated as a sine.